sdk/helper/pluginutil/identity_token.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package pluginutil
+
+import (
+	"time"
+)
+
+type IdentityTokenRequest struct {
+	// Audience identifies the recipient of the token
+	Audience string
+	// TTL is the duration that the token will be valid for
+	TTL time.Duration
+}
+
+type IdentityTokenResponse struct {
+	// Token is the plugin identity token
+	Token IdentityToken
+	// TTL is the capped duration that the token is valid for
+	TTL time.Duration
+}
+
+type IdentityToken string
+
+// String returns a redacted token string. Use the Token() method
+// to obtain the non-redacted token contents.
+func (t IdentityToken) String() string {
```

Comment:
Can add tests for these if folks think it's warranted :)

Reply:
I think these structs aren't needed for the SDK field helpers? I need them in a PR I'm opening so you can remove it from this PR :)

Reply:
Yup, they shouldn't be needed for the field helpers! Since they were part of the SDK, I added them in as part of all sdk helpers we'd need 😅  makes sense to remove it from this PR if it's being included elsewhere though, I can do that 👍🏼 Thanks!

Reply:
Perfect, thanks @vinay-gopalan!